.clang-format

```diff
@@ -0,0 +1,13 @@
+﻿---
```

Comment:
This should closely match the style for most files. Calling clang-format for new files will automatically pick this up.